pkg/hooks/README.md

```diff
@@ -44,7 +44,7 @@ Each JSON file should contain an object with the following properties:
         Entries MUST be [POSIX extended regular expressions][POSIX-ERE].
     * **`hasBindMounts`** (OPTIONAL, boolean) If `hasBindMounts` is true and the caller requested host-to-container bind mounts (beyond those that CRI-O or libpod use by default), this condition matches.
 * **`stages`** (REQUIRED, array of strings) Stages when the hook MUST be injected.
-    Entries MUST be chosen from the 1.0.1 OCI Runtime Specification [hook stages][spec-hooks].
+    Entries MUST be chosen from the 1.0.1 OCI Runtime Specification [hook stages][spec-hooks] or from extention stages supported by the package consumer.
```

Comment:
Being strict here helps catch typos like `pre-start` (where the user likely intended `prestart`).  But it's going to make sharing a single `/usr/share/containers/oci/hooks.d`.  Do we expect all tools consuming that directory to always agree on the set of extension stages?  Do we expect tools to add additional directories for extension stages not yet adopted across the whole consumer community?  Do we want to provide a way for tools to allow unrecognized extension stages?

Reply:
I'm content to say that all users of a single `hooks.d` should share the same extension stages - simplest solution for now. We can make further changes later if this proves inadequate.

Reply:
> I'm content to say that all users of a single hooks.d should share the same extension stages - simplest solution for now.

So what directory should I change to when I add `postexit` support for #730?  Does the config need to be per-directory to avoid breaking users who don't need `postexit` but have existing content in `/usr/share/containers/oci/hooks.d`?